app/models/condition.rb

```diff
@@ -86,10 +86,7 @@ def self.evaluate(cond, rec, inputs={}, attr=:expression)
 
   def self.do_eval(expr)
     if expr =~ /^__start_ruby__\s?__start_context__\s?(.*)\s?__type__\s?(.*)\s?__end_context__\s?__start_script__\s?(.*)\s?__end_script__\s?__end_ruby__$/im
-      context, col_type, script = [$1, $2, $3]
-      context = MiqExpression.quote(context, col_type)
-      result = SafeNamespace.eval_script(script, context)
-      raise "Expected return value of true or false from ruby script but instead got result: [#{result.inspect}]" unless result.kind_of?(TrueClass) || result.kind_of?(FalseClass)
+      raise "Ruby script in expression is no longer supported"
```

Comment:
The response to the community to date is that RegEx feature in conditions should cover off all advanced use cases for filtering.

My assumption was that it would no longer be available in the UI as an option in the dropdown.

@Fryguy what was the alternative?

thanks